server/providers/couchdb.js

```diff
@@ -250,42 +250,6 @@ const couchDb = (app, io, pouchOptions) => {
       .catch(() => res.send([]))
   })
 
```

Comment:
hello @lilitkarapetyan - then ext method `/turns` doesn't seem related to mapping.  Are you sure it's right to delete it?